webrtc-stats.html

```diff
@@ -1660,6 +1661,20 @@
                   Only present for inbound audio tracks. The total number of inbound audio samples
                   that are concealed samples. A concealed sample is a sample that is based on data
                   that was synthesized to conceal packet loss and does not represent incoming data.
+                  This includes <a>concealedAudibleSamples</a>.
+                </p>
+              </dd>
+              <dt>
+                <dfn><code>concealedAudibleSamples</code></dfn> of type <span class="idlMemberType">
+                <a>unsigned long long</a></span>
+              </dt>
+              <dd>
+                <p>
+                  Only present for inbound audio tracks. The total number of concealed audio samples
+                  (see <a>concealedSamples</a>) that was played out during an audible portions of the
+                  stream. Audible means that the received audio is not considered background noise or
+                  silence by the user agent. It is up to the implementation to determine what is
+                  considered background noise.
```

Comment:
add: The value is set to undefined for user-agents that do not implement this metric.